ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -175,120 +146,61 @@ S3SourcePath(const std::string &dir, const std::string &name)
     return (dir + "/" + strippedName);
 }
 
-// Return if the file exists. First checks the cache, and then the S3 Bucket.
+// Return if the file exists, via looking into the S3 Bucket. Also fetches the file size if exists.
```

Comment:
"Also fetches the file size if exists." -> "Also, if the file exists, fetch the file size."

Reply:
changed [fa8365c](https://github.com/wiredtiger/wiredtiger/pull/9708/commits/fa8365ce9ebfbe01a182b327672b7475457c9f95)